test/test_tensorexpr.py

```diff
@@ -1295,6 +1295,21 @@ def bias_gelu(bias, y):
             x = warmup_and_run_forward(traced, a, b)
             self.assertLastGraphAllFused()
 
+    def test_exp_pow(self):
+        devices = ["cuda", "cpu"] if torch.cuda.is_available() else ["cpu"]
+
+        @torch.jit.script
+        def do_exp(x, y, z):
+            return ((x * y) * 2) * torch.pow(z, 2)
```

Comment:
Nit: can the expression be simplified? I guess what really matters here is just `torch.pow(z, 2)`?

Reply:
It could be, but this is equivalent to the repro in the bug report.